### PR TITLE
add lint

### PR DESCRIPTION
**Pull request by cw-ozaki**
Added chart lint.

rules:
- `nindent` check
- helm lint
    - https://helm.sh/docs/helm/helm_lint/
- schema check (kubeval)
    - https://github.com/instrumenta/kubeval 

#### Checklist

- [X] Chart Version bumped

### CODE REVIEW

File: aws-alb-ingress-controller/Makefile [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
 
+.PHONY: lint
+lint: lint-default
+
+.PHONY: lint-default
+lint-default:
+	@echo "=> Indent check default values.yaml"
+	../.circleci/lint-chart.sh $$PWD/templates
+	@echo "=> Linting default values.yaml"
+	helm lint --strict
```
Comment (cw-ozaki) [inline aws-alb-ingress-controller/Makefile:16]: This lint depends on the version of helm
https://github.com/helm/helm/blob/fee2257e3493e9d06ca6caa4be7ef7660842cbdb/pkg/lint/rules/deprecations.go

File: aws-alb-ingress-controller/Makefile [hunk 2]
```diff
@@ -1,9 +1,23 @@
+KUBERNETES_VERSION="1.18.1"
```
Comment (cw-ozaki) [inline aws-alb-ingress-controller/Makefile:1]: The schema version used by kubeval is only `1.18.1`
https://github.com/instrumenta/kubernetes-json-schema

**Comment by cw-ozaki** [comment]
@cw-sakamoto @taishin 
Are there any rules that you would like to lint in making a helm chart?

**Review by taishin — APPROVED** [review]

**Review by cw-sakamoto — APPROVED** [review]
At this change, we need to write k8s version in each Makefile for chart, how do you think for maintenance it ?

**Comment by cw-ozaki** [comment]
@cw-sakamoto 
```
find . -name Makefile | xargs sed 's/KUBERNETES_VERSION="1.18.1"/KUBERNETES_VERSION="next version"/g'
```

**Comment by cw-ozaki** [comment]
Oops, kubeval didn't have the schema for 1.19 and 1.20...
https://github.com/instrumenta/kubernetes-json-schema

**Comment by cw-ozaki** [comment]
@cw-sakamoto 
[instrumenta/kubernetes-json-schema](https://github.com/instrumenta/kubernetes-json-schema) forked because the new schema is not updated.
https://github.com/cw-ozaki/kubernetes-json-schema
Only the latest schemas are always added in this repository.

**Review by cw-sakamoto — APPROVED** [review]

**Pull request merged**